### DISCUSSION TITLE
File path does not change for certain shapefiles after exporting

### DISCUSSION THREAD

**Discussion by oliverL95** [opengisch/QField#4769] [Q&A]
Having a problem getting certain layers to load on my mobile device. I am using QGIS 3.340 and Qfield 4.6.0. I have 4 shapefiles and 1 raster I am trying to load onto my mobile device. 3 of the shapefiles won't load and give me an error. When I open the Qfield version of the project on my desktop I notice that the file path of the shapefile that works is to the new Qfield export folder I made. The file path for the other 3 shapefiles is to the original folder that the desktop QGIS project is in, which is obviously not on my mobile so it can't load the layers. I made sure all the layers are set to "copy" during cable export, and that save paths is set to "relative", but the problem persists. 

Any ideas on how to fix it, or at least a work around, to manually change the file paths in the Qfield project?

Thanks

**Comment by oliverL95** [comment]
I figured out a temporary work around, which works for me and my 5 layers, but would be hell if you had more. If you open the Qfield project on your desktop, and open the Qfield export folder, you can drag and drop the shapefiles into the project, and then copy the style from the layer in the project and remove the layer with the wrong file path. Then save the project and copy the entire folder onto the mobile device. This suggests that the files are being written in the right spot, so I think its a problem with some Qfield settings

**Reply by suricactus** [reply]
I guess your files were imported with absolute and/or are outside the .qgs project file directory.

**Reply by oliverL95** [reply]
I received a reply from Qfield. Their workaround is to use geopackages instead of shapefiles, or better yet, don't use the qfield sync at all. As of a recent updated, the qfield app can open the normal qgis project, so you can just drag and drop the entire project onto your mobile device and not have to create an export folder
